src/model/keychain.ts

```diff
@@ -87,19 +112,21 @@ export async function remove(key) {
   }
 }
 
-export async function loadAllKeys(authenticationPrompt) {
+export async function loadAllKeys(): Promise<null | UserCredentials[]> {
```

Comment:
btw this function no longer needs to be exported

Reply:
It's still used in SettingsModal